RFC/src/RFC-0241_AtomicSwapXMR.md

```diff
@@ -0,0 +1,830 @@
+b_{Sra} - b_{Sra}' &= x_a \\\\
+\end{aligned}
+\tag{22}
+$$
+
+
+### The lapse transaction
+
+If something goes wrong and Alice never publishes her refund transition, Bob needs to wait for the
+lock height `height_2` to pass. This will allow him to create the lapse transaction to claim the Tari. 
+
+Bob constructs the lapse transaction with
+$$
+\begin{aligned}
+a_{Slb} &= r_{Slb_a} +  e_l(v_{i}) \\\\
+b_{Slb} &= r_{Slb_b} + x_b + e_l(k_{Slb} + k_i) \\\\
+e_l &= \hash{ (R_{Sl} + (X_b)) \cat \alpha_i \cat \input_i \cat (K_{Sla} + K_{Slb}) \cat C_i} \\\\
+a_{Sl} &= a_{Sla} + a_{Slb} \\\\
+b_{Sl} &= b_{Sla} + b_{Slb} \\\\
+R_{Sl} &= r_{Slb_a} \cdot H + r_{Slb_b} \cdot G + R_{Sla}\\\\
+a_{Mlb} &= r_{Mlb_a} + e(v_{l}) \\\\
+b_{Mlb} &= r_{Mlb_b} + e(k\_{Olb}+k_l) \\\\
+R_{Mlb} &= a_{Mlb} \cdot H + b_{Mlb} \cdot G \\\\
+e &= \hash{ (R_{Mla} + R_{Mlb}) \cat \script_l \cat F_l \cat (K_{Ola} + K_{Olb}) \cat C_l} \\\\
+R_{Ml} &= R_{Mla} + R_{Mlb} \\\\
+\so_{lb} &= k_{Slb} - k_{Olb} \\\\
+\so_{r} &= \so_{la} +\so_{lb} \\\\
+\end{aligned}
+\tag{23}
+$$
+
+This allows Bob to claim the Tari he originally wanted, but it also exposes his Monero key \\(x_b\\)
+This means if Alice ever comes back online, she can calculate \\(X\\) and claim the Monero she wanted all along using:
+$$
+\begin{aligned}
+b_{Sl} &= b_{Slb} + b_{Slb} \\\\
+b_{Sl} - b_{Slb} &= b_{Slb} \\\\
+b_{Slb} &= r_{Slb_b} + x_a + e_r(k_{Slb} + k_i) \\\\
+b_{Slb} - b_{Slb}' &= r_{Slb_b} + x_b + e_r(k_{Slb} + k_i) -(r_{Slb_b} +  e_r(k_{Slb}+k_i))\\\\
+b_{Slb} - b_{Slb}' &= x_b \\\\
+\end{aligned}
+\tag{24}
+$$
+
+## Notation
+
+Where possible, the "usual" notation is used to denote terms commonly found in cryptocurrency literature. Lower case 
+characters are used as private keys, while uppercase characters are used as public keys. New terms introduced here are 
+assigned greek lowercase letters in most cases. Some terms used here are noted down in [TariScript]. 
+
+| Name                        | Symbol                | Definition |
+|:----------------------------|-----------------------| -----------|
+| subscript s                 | \\( _s \\)            | The swap transaction |
+| subscript r                 | \\( _r \\)            | The refund transaction |
+| subscript l                 | \\( _l \\)            | The lapse transaction |
+| subscript a                 | \\( _a \\)            | Belongs to Alice |
+| subscript b                 | \\( _b \\)            | Belongs to Bob |
+| Monero key                  | \\( X \\)             | Aggregate Monero public key on Ristretto |
+| Alice's Monero key          | \\( X_a \\)           | Alice's partial  Monero public key on Ristretto |
+| Bob's Monero key            | \\( X_b \\)           | Bob's partial  Monero public key on Ristretto   |
+| Monero key                  | \\( Xm \\)            | Aggregate Monero public key on ed25519 |
+| Alice's Monero key          | \\( Xm_a \\)          | Alice's partial  Monero public key on ed25519 |
```

Comment:
```suggestion
| Alice's Monero key          | \\( Xm_a \\)          | Alice's partial  Monero public key on Curve25519 |
```
ed25519 actually refers to an EdDSA signature on Curve25519

Reply:
this might actually be fine as the key in this case could be considered a signature?